supplements/waterdeep-dungeon-of-the-mad-mage/lycanthrope-werebat.xml

```diff
@@ -0,0 +1,154 @@
+		<rules>
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_BLOOD_DRINKERS" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_RESISTANCE" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_DEXTERITY" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_SPEED" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_SHAPECHANGER" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_ECHOLOCATION" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_KEEN_HEARING" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_NIMBLE_ESCAPE" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_SUNLIGHT_SENSITIVITY" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_MULTIATTACK" />
+			<grant type="Racial Trait" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_WEAPONS" />
+		</rules>
+	</element>
+	
+<!-- Blood Drinkers -->
+    <element name="Blood Drinkers" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_BLOOD_DRINKERS">
+        <description>
+            <p> A werebat must consume at least 1 pint of fresh blood each night, or it weakens and gains one level of exhaustion that no amount of rest alone can remove. Each pint of blood the werebat consumes removes one level of exhaustion gained in this fashion.</p>
+        </description>
+        <sheet>
+            <description> A werebat must consume at least 1 pint of fresh blood each night, or it weakens and gains one level of exhaustion that no amount of rest alone can remove. Each pint of blood the werebat consumes removes one level of exhaustion gained in this fashion.</description>
+        </sheet>
+    </element>
+<!-- Resistance -->
+    <element name="Werebat Resistance" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_RESISTANCE">
+		<description>
+			<p>You are immune to bludgeoning, piercing, and slashing damage from non magical weapons that aren't silvered in Raven or Hybrid Form.</p>
+		</description>
+		<sheet usage="Raven or Hybrid Form">
+			<description>You are immune to bludgeoning, piercing, and slashing damage from non magical weapons that aren't silvered.</description>
+		</sheet>
+	</element>
+<!-- Dexterity -->
+    <element name="Werebat Dexterity" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_DEXTERITY">
+		<description>
+			<p>A player character who becomes a werebat gains a Dexterity of 17 if their score isn't already higher.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<stat name="dexterity:score:set" value="17" bonus="base" />
+		</rules>
+	</element>
+<!-- Speed -->
+    <element name="Werebat Speed" type="Racial Trait" source="Waterdeep: Dungeon of the Mad Mage" id="ID_WOTC_WDMM_RACIAL_TRAIT_WEREBAT_SPEED">
```

Comment:
Is the code deletion here because of the stat block I'll be adding? Just want to verify.

Reply:
No, i removed Dexterity since it's now added from the main Trait. 
I removed Blood Drinkers since i don't believe it counts as Trait either (even though i personally would count it as such). I kinda assume that if it isn't added into stat-block's trait part, then it doesn't count as trait granted by the lycantropy... at least RAW, but hard to say if that's the case, they could count those parts of monster description/lore as traits.

Reply:
It even sounds unlike other traits, since it uses plural "Drinkers".